community_supplied/Interface/snmp-interface-throughput.rule

```diff
@@ -1,11 +1,282 @@
+                    }
+                }
+                type float;
+                description "ML predicted output bandwidth utilization";
+            }
+            /*
+            * Fields defined with variables and constant values
+            */
+            field threshold {
+                constant {
+                    value "{{threshold}}";
+                }
+                type float;
+                description "Interface bandwidth utilization increase threshold";
+            }
+            /*
+            * Anomaly detection logic.
+            */
+            trigger in-bw-utilization {
+                synopsis "Interface input traffic kpi";
+                description "Sets health based on input traffic exceed threshold";
+                frequency 180s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * the input throughput utilization ($in-bw-util %) is greater than
+                * the threshold ($threshold %).
+                */
+                term is-input-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$in-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name input throughput utilization ($in-bw-util %) crossed threshold value ($threshold %). Mbps:$in-mbps Bps:$input-bytes bps";
+                        }
+                    }
+                }
+                /*
+                * Defaults color to green.
+                */
+                term is-input-bandwidth-normal {
+                    then {
+                        status {
+                            color green;
+                            message "$interface-name input throughput utilization ($in-bw-util %) is normal. Mbps:$in-mbps Bps:$input-bytes bps ";
+                        }
+                    }
+                }
+            }
+            trigger out-bw-utilization {
+                synopsis "Interface output traffic kpi";
+                description "Sets health based on output traffic exceed threshold";
+                frequency 180s;
+                /*
+                * Sets color to red and sends out an anomaly notification when
+                * the output throughput utilization ($out-bw-util %) is greater
+                * than the threshold ($threshold %).
+                */
+                term is-output-bandwidth-abnormal {
+                    when {
+                        greater-than-or-equal-to "$out-bw-util" "$threshold";
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "$interface-name output throughput utilization($out-bw-util%) crossed threshold value ($threshold %). Mbps:$out-mbps Bps:$output-bytes bps";
```

Comment:
it (Bps:$output-bytes bps) is bytes not bps hence same. cane be removed

Reply:
Addressed and changed the message from "Bps:$output-bytes bps" to "Output-bytes:$output-bytes"